kotlin-styled-next/src/main/kotlin/styled/InlineStyles.kt

```diff
@@ -49,3 +45,32 @@ fun StyledElement.toStyle(prefix: Boolean = true): Any {
 
     return prefixed
 }
+
+private fun CssDeclarations.mapToObj(): dynamic {
+    val res = js { }
+    forEach { (key, value) ->
+        res[key] = when (value) {
+            is String, is Number -> value
+            else -> value.toString()
+        }
+    }
+    return res
+}
+
+internal fun CssDeclarations.buildPrefixedString(): String {
+   val res = mapToObj()
+
+    val prefixed = prefix(res) as Object
+    return buildString {
+        Object.keys(prefixed).forEach {
+            val value = prefixed.asDynamic()[it]
+
+            if (value is JsArray) {
+                val displayValue = value.find(js("function(element) { return !element.startsWith('-') }"))
```

Comment:
What about the other values?

Reply:
In inline prefixer above there are gets thrown away too with the comment
`Find the unprefixed value in the array and use it: multiple values don't work for some reason`. Can duplicate comment here

And as I know `styled-components` do not use multiple values for a property either